# HG changeset patch
# Parent 0dd74b0bb243a81f78031471b9f12153542fecf3
# User sbr
# Date 1349834478 -7200

Fix (r21933): Add space for the 'current order'-double arrow in the timetable gui.

diff -r 0dd74b0bb243 src/timetable_gui.cpp
--- a/src/timetable_gui.cpp	Tue Oct 09 20:59:29 2012 +0000
+++ b/src/timetable_gui.cpp	Wed Oct 10 05:08:47 2012 +0200
@@ -369,7 +369,7 @@
 
 				bool rtl = _current_text_dir == TD_RTL;
 				SetDParam(0, 99);
-				int index_column_width = GetStringBoundingBox(STR_ORDER_INDEX).width + GetSpriteSize(rtl ? SPR_ARROW_RIGHT : SPR_ARROW_LEFT).width + 3;
+				int index_column_width = GetStringBoundingBox(STR_ORDER_INDEX).width + 2 * GetSpriteSize(rtl ? SPR_ARROW_RIGHT : SPR_ARROW_LEFT).width + 3;
 				int middle = rtl ? r.right - WD_FRAMERECT_RIGHT - index_column_width : r.left + WD_FRAMERECT_LEFT + index_column_width;
 
 				const Order *order = v->GetOrder(order_id);
